cmake/CMakeSettings.json

```diff
@@ -16,6 +16,11 @@
           "value": "True",
           "type": "BOOL"
         },
+        {
+          "name": "onnxruntime_BUILD_WINML_IN_MICROSOFT_NS",
+          "value": "True",
+          "type": "BOOL"
+        },
```

Comment:
Here too

Reply:
code secretly not saving my files... done..